common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3748,6 +3748,12 @@
     },
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_ENCODER" : {
```

Comment:
I think we are dealing with invalid encoder not some unsupported one. Could you rename it to: `INVALID_EXPRESSION_ENCODER`